### PR TITLE
Move XTAP APIs to v1.0

### PR DESCRIPTION
**Pull request by jkdouglas**
v1.0 prep

### CODE REVIEW

**Comment by jkdouglas** [comment]
@FaithOmbongi - this is a copy/paste from our current Beta docs. Hopefully the review should be light. I don't think we have a CSDL PR yet, but as soon as we do I will link it here.

**Comment by jkdouglas** [comment]
@FaithOmbongi - this is a light PR to bring the XTAP beta docs over to v1.0. Can you please review at your convenience?

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -1,5 +1,87 @@
...
+        },
+        {
+          "Id": "5c4ea04c-ccb5-4d1b-bf09-7b29a39710ad",
+          "ApiChange": "Resource",
+          "ChangedApiName": "crossTenantAccessPolicyTargetConfiguration",
+          "ChangeType": "Addition",
+          "Description": "Added the [crossTenantAccessPolicyTargetConfiguration](https://docs.microsoft.com/en-us/graph/api/resources/crosstenantaccesspolicytargetconfiguration?view=graph-rest-1.0) resource.",
+          "Target": "crossTenantAccessPolicyTargetConfiguration"
+        },
+        {
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:61]: Add an entry for tenantRelationshipAccessPolicyBase resource type.

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @jkdouglas - I've gone with the assumption that the v1.0 will be exactly as the beta API except the allowedCloudEndpoints property. I hope my assumption is right.

2 gaps:

- Add a changelog entry for tenantRelationshipAccessPolicyBase.
- Document the crossTenantAccessPolicy relationship in the policyroot.md file and add a related changelog entry.

**Comment by jkdouglas** [comment]
> Hi @jkdouglas - I've gone with the assumption that the v1.0 will be exactly as the beta API except the allowedCloudEndpoints property. I hope my assumption is right.
> 
> 2 gaps:
> 
> * Add a changelog entry for tenantRelationshipAccessPolicyBase.
> * Document the crossTenantAccessPolicy relationship in the policyroot.md file and add a related changelog entry.

@FaithOmbongi - could you review the 2 changes you requested and make sure I covered them? I added changelogs for each and added the relationship for crossTenantAccessPolicy in policyroot.

File: api-reference/v1.0/toc.yml [hunk 2]
```diff
@@ -5138,6 +5138,43 @@ items:
                   href: api/ipnamedlocation-update.md
                 - name: Delete
                   href: api/ipnamedlocation-delete.md
+                - name: Cross-tenant access (preview)
```
Comment (Lauragra) [inline api-reference/v1.0/toc.yml:5141]: Why do we have this entry with (preview) appended in the v1.0 TOC? I think we need to delete line 5141. Also, please remove the (preview) from the Cross-tenant access node in the toc.yml file in the /beta folder,

Comment (jkdouglas) [reply api-reference/v1.0/toc.yml]: I think it was a typo. I went ahead and removed the (preview) tag and updated the TOC for both v1.0 and beta.

**Pull request merged**